apps/wallet/src/ui/app/pages/home/nfts/index.tsx

```diff
@@ -57,11 +58,17 @@ function NftsPage() {
 				) : null}
 				{nfts?.length ? (
 					<div className="grid w-full grid-cols-2 gap-x-3.5 gap-y-4">
-						{nfts.map(({ objectId }) => (
+						{nfts.map(({ objectId, type }) => (
 							<Link
 								to={`/nft-details?${new URLSearchParams({
 									objectId,
 								}).toString()}`}
+								onClick={() => {
+									ampli.viewedCollectible({
```

Comment:
Mmm I messed this event name up while adding in some events yesterday, I should rename this to `clickedCollectibleCard` to follow the general pattern I've been going for.

To answer your broader question about logging events via effects (e.g. `viewed collectible page` vs. `clicked collectible card`), the Amplitude folks recommended more action-based logging instead of wiring everything up as "viewed XXX" _in most cases_ (it's okay to track page views for things we really care about). So clicking the collectible card implies a page view here